p/api/fever.php

```diff
@@ -69,7 +69,7 @@ final class FeverDAO extends Minz_ModelPdo
 	 */
 	private function bindParamArray(string $prefix, array $values, array &$bindArray): string {
 		$str = '';
-		for ($i = 0; $i < count($values); $i++) {
+		for ($i = 0, $iMax = count($values); $i < $iMax; $i++) {
```

Comment:
Please refrain from making unrelated changes, which complicate the review

Reply:
[EA] 'count(...)' is used in a loop and is a low performing construction

Reply:
If you accept, i can move max before the loop

Reply:
The problem is that those random changes are unrelated to the topic of the PR, which leads to lots of noise and inefficient PRs

Reply:
done